h2o-py/h2o/estimators/pca.py

```diff
@@ -269,3 +269,15 @@ def max_runtime_secs(self, max_runtime_secs):
         self._parms["max_runtime_secs"] = max_runtime_secs
 
 
+
+    def transform(self, X, y=None, **params):
+        """
+        Transform the given H2OFrame with the fitted PCA model.
```

Comment:
Maybe we can state the purpose of the method? that it is used in sklearn pipelines?

Reply:
Yes, that is a good point. I changed it.